nameserver/zone.go

```diff
@@ -121,33 +121,37 @@ func (zone *ZoneDb) AddRecord(ident string, name string, ip net.IP) error {
 	return nil
 }
 
-func (zone *ZoneDb) DeleteRecord(ident string, ip net.IP) error {
+func (zone *ZoneDb) DeleteRecords(ident string, name string, ip net.IP) (count int) {
```

Comment:
don't use a named return value here. It saves you typing five characters later, which isn't worth the obscurity.